### PR TITLE
ovn-kubernetes-ga

### PR DESCRIPTION
**Pull request by pecameron**
Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@squeed @dcbw @knobunc PTAL

File: enhancements/20190919-OVN-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: ovn-kubernetes-ga
```
Comment (deads2k) [inline enhancements/20190919-OVN-Kubernetes.md]: update PR title to match please

Comment (pecameron) [reply enhancements/20190919-OVN-Kubernetes.md]: @deads2k done

File: enhancements/20190919-OVN-Kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+#### Node-level
+The node-level components are configured as a daemonset. They run on all
+nodes, including the control-plane. They watch for endpoint changes,
+watch the sbdb for changes, and configure the nodes.
+
+### Test Plan
+OVN-Kubernetes must pass all standard QE tests, as well as the full e2e
+suite. It does not need any special testing, as the GA release does not
+enable any new features.
```
Comment (smarterclayton) [inline enhancements/20190919-OVN-Kubernetes.md]: I don’t think this is sufficient.  I’d expect to see a full test suite verifying the failover and recovery of the OVN control plane in a number of failure modes that give confidence of this somewhat novel architecture.

That suite would be required to be stable and comprehensive prior to GA

File: enhancements/20190919-OVN-Kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+### Graduation Criteria
+GA:
+- OVN-Kubernetes is stable
+- The control-plane is low-latency: changes are installed on the nodes in under 10 seconds
```
Comment (smarterclayton) [inline enhancements/20190919-OVN-Kubernetes.md]: I need better constraints than this.  Document in terms of p99 and p90, expected behavior on small systems

File: enhancements/20190919-OVN-Kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+- Reliable control plane: losing a master node does not cause more than a few seconds of extra latency, and no loss of network connectivity
```
Comment (smarterclayton) [inline enhancements/20190919-OVN-Kubernetes.md]: Latency for what?  Changes?  Dataplane traffic?  New connections?  Clarify here

**Pull request merged**